nvflare/app_common/aggregators/intime_accumulate_model_aggregator.py

```diff
@@ -15,6 +15,7 @@
 from typing import Any, Dict, Union
 
 from nvflare.apis.dxo import DXO, DataKind, from_shareable
+from nvflare.apis.event_type import EventType
```

Comment:
Why do we need to modify this?  It doesn't seem to be related to this PR.

Reply:
The old implementation changed the aggregation_weights, exclude_vars, expected_data_kind parameters in the object constructor. When using the inspect to parse the object attributes won't be able to figure out what parameters have been used to create the object, because the parameters have been changed to a different format.

Also performing the data conversion during the START_RUN event is the proper way of implementation.